app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -153,6 +177,20 @@ export default class CinematicLankBoss {
     })
   }
 
+  /**
+   * Plays an action on a lank. If you want an animation to loop, and it isn't,
+   * you need to make the action loop on the ThangType.
+   * @param {string} key The lanks unique key
+   * @param {string} action The action to queue onto the lank
+   */
+  playActionOnLank (key, action) {
+    const lank = this.lanks[key]
+    if (!lank) {
+      return console.warn(`Tried to play action '${action}' on non existant lank '${key}'`)
```

Comment:
Do we want to know when this happens?  Should this be sent to something like sentry?  If so maybe we should log an error or throw an error that can be caught elsewhere and reported.

Reply:
Also - we should split these out onto two different lines.  Console.warn may return undefined but we should be explicit about what we are returning.

Reply:
Because the cinematic designer can technically make the lank vanish before the action is played I don't want the result to be too devastating.

With more time I want to create a cinematic verifier mode that plays the cinematics quickly and presents warnings and errors to the designer.

Reply:
Split into 2 lines and added to doc.